docs/12-performance-test.md

```diff
@@ -3,84 +3,64 @@ Performance tests
 
 The following notes describe how to conduct some manual tests of performance to help understand whether changes to the
 code have increased or decreased the performance. These tests are based on the system tests described in
-[09-dev-guide#System-tests](09-dev-guide.md#System-tests). Note that currently the following needs to be run on
-an x86 machine.
+[09-dev-guide#System-tests](09-dev-guide.md#System-tests).
 
-Under `scripts/test` we have system tests for deploying everything, and for compaction performance testing. The
-compaction tests take control of when compactions run in order to produce more deterministic results. The tests for
-deploying everything test direct ingest, while the compaction performance tests currently use a queue.
+Under `scripts/test` we have system tests for testing performance of compaction and bulk import.
+The performance and functional tests take control of when compactions run in order to produce more deterministic results.
 
-## Deploy all
+## Compaction performance
+
+This will test the performance of standard ingest, standard compactions and compaction job creation. It will not 
+perform any partition splitting. This will take around an hour. This is intended to avoid any variance that may be 
+caused by the number of input files or the amount of data processed at once.
+
+There are a variety of scenarios that can occur when compaction is performed on scheduled jobs (as in the deploy all 
+system test, or normal system functioning). The compaction performance test avoids this by disabling the scheduled jobs 
+and triggering those processes directly.
```

Comment:
There was a bit more information before about situations where compactions can run at odd times. I'm not sure if this is clear about what we're trying to avoid by triggering things directly.